woocommerce-services.php

```diff
@@ -538,7 +539,7 @@ public function rest_api_init() {
 			$rest_account_settings_controller->register_routes();
 
 			require_once( plugin_basename( 'classes/class-wc-rest-connect-services-controller.php' ) );
-			$rest_services_controller = new WC_REST_Connect_Services_Controller( $this->api_client, $settings_store, $logger, $schemas_store );
+			$rest_services_controller = new WC_REST_Connect_Services_Controller( $this->api_client, $settings_store, $logger, $schemas_store, $this->nux );
```

Comment:
It looks like the addition of `WC_Connect_Nux` was missed in the `WC_REST_Connect_Services_Controller` class.